server/sonar-server/src/main/java/org/sonar/server/qualityprofile/ws/SetDefaultAction.java

```diff
@@ -65,15 +67,17 @@ public void handle(Request request, Response response) {
     QProfileReference reference = QProfileReference.from(request);
     try (DbSession dbSession = dbClient.openSession(false)) {
       QualityProfileDto qualityProfile = qProfileWsSupport.getProfile(dbSession, reference);
+      OrganizationDto organization = dbClient.organizationDao().selectByUuid(dbSession, qualityProfile.getOrganizationUuid())
+        .orElseThrow(() -> new IllegalStateException(format("Cannot find organization '%s' for quality profile '%s'", qualityProfile.getOrganizationUuid(), qualityProfile.getKee())));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 183 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)